nbt/src/main/java/net/kyori/adventure/nbt/ListBinaryTag.java

```diff
@@ -366,6 +366,62 @@ default double getDouble(final @NonNegative int index, final double defaultValue
     return defaultValue;
   }
 
+  /**
+   * Gets a list.
+   *
+   * @param index the index
+   * @return the list, or an empty list
```

Comment:
when is the empty list returned? when `index` is out of bounds? or when the tag at `index` is not a list tag? if it's the second, what happens when `index` is out of bounds?